robottelo/constants.py

```diff
@@ -797,8 +797,6 @@
     'Edit hosts',
     'Edit partition tables',
     'Red Hat Access Logs',
-    'Remote Execution Manager',
-    'Remote Execution User',
```

Comment:
I think so, according to [BZ1322392](https://bugzilla.redhat.com/show_bug.cgi?id=1322392).